src/components/FilterBar/index.js

```diff
@@ -0,0 +1,21 @@
+import Link from 'next/link';
+
+import StyledButton from '../Button/styled';
+
+import StyledFilterBar from './styled';
+
+export default function FilterBar() {
+	return (
+		<StyledFilterBar>
+			<Link href="./coffee">
+				<StyledButton variant="coffee">Coffee</StyledButton>
+			</Link>
+			<Link href="./tea">
+				<StyledButton variant="tea">Tea</StyledButton>
+			</Link>
+			<Link href="./">
+				<StyledButton variant="all">Show All</StyledButton>
+			</Link>
```

Comment:
Please never ever nest `a` and `button` elements in HTML. They have two separate functionalities and it is not good practice to nest those. Please style the link as a button but don't use a `button` element.